tests/testthat/_snaps/bs.md

```diff
@@ -1,3 +1,13 @@
+# check_name() is used
+
+    Code
+      prep(rec, training = dat)
+    Condition
+      Error in `step_bs()`:
+      Caused by error in `bake()`:
```

Comment:
Do we want this to say `bake()` or `prep()` (since the user calls `prep()` here)?

Reply:
I think this would be really hard to do, since `prep(retain = TRUE)` applies `bake()`.

so basically, this is an error from `bake()`, because bake by default is run `prep()` is called.

kinda messy, but I don't see a way to neatly fix it